### PR TITLE
adapter: work-around missing res.response

### PR DESCRIPTION
**Pull request by sam-github**
fix https://github.com/strongloop/strong-supervisor/issues/213

### CODE REVIEW

File: lib/adapter.js [hunk 1]
```diff
@@ -377,7 +377,7 @@ function createMessageListener() {
 function createProbeListener(probeName) {
   monitor.on(probeName, function(res) {
     var duration = 0;
-    if (probeName === 'express') {
+    if (probeName === 'express' && res.response) {
```
Comment (sam-github) [inline lib/adapter.js]: the intention of the code is that there is only a `res.response.duration` for express probes, I've no reason to think any other probes have one, and I've no idea why the express probe doesn't always have one. This PR is a work-around, we can figure out what is really broken next week - but it doesn't effect apiconnect, which doesn't depend on this metric anyway.

**Comment by kjdelisle** [comment]
LGTM

**Comment by kjdelisle** [comment]
@slnode ok to test

**Comment by kjdelisle** [comment]
@slnode test please

**Comment by kjdelisle** [comment]
@slnode test please

File: lib/adapter.js [hunk 2]
```diff
@@ -377,7 +377,7 @@ function createMessageListener() {
 function createProbeListener(probeName) {
   monitor.on(probeName, function(res) {
```
Comment (sam-github) [inline lib/adapter.js]: I think the argument name is wrong here, this is the `data` object from the `express` even that had docs removed in https://github.com/RuntimeTools/appmetrics/pull/260/files, right? `res` makes it seem like it is a https://nodejs.org/docs/latest/api/http.html#http_class_http_serverresponse

But the express probe got removed, and replace with strong-express-metrics... and now we are hitting this, which means that there is still an `express` event, what is the structure of this event?

File: lib/adapter.js (hunk 1 above)
Comment (sam-github) [inline lib/adapter.js]: so the `res.response` is only relevant to code that was deleted

**Pull request merged**